nengo/spa/actions.py

```diff
@@ -154,7 +154,11 @@ class Actions(object):
     def __init__(self, *args, **kwargs):
         self.actions = None
         self.args = args
-        self.kwargs = kwargs
+        self.kwargs = OrderedDict(kwargs)
```

Comment:
It's probably good to remember what order people added actions in, so I support the use of `OrderedDict`. If that's something that's going to effect the model that's built, then we definitely want to ensure it's the same between Python versions.